packages/frontend/src/components/DashboardTiles/AddTileButton.tsx

```diff
@@ -93,6 +98,13 @@ const AddTileButton: FC<Props> = ({ onAddTiles, intent, popoverPosition }) => {
                                                 ),
                                             );
                                         }
+                                        sessionStorage.setItem(
+                                            'hasDashboardChanges',
+                                            JSON.stringify(
```

Comment:
do we need to stringify boolean values?

Reply:
it wasn't catching it when it says 'false' as a string